ramalama/common.py

```diff
@@ -273,18 +253,16 @@ def load_cdi_config(spec_dirs: list[str]) -> CDI_RETURN_TYPE | None:
                 if ext in [".yaml", ".yml"]:
                     try:
                         with open(file_path, "r") as stream:
-                            return load_cdi_yaml(stream)
-                    except OSError:
+                            return yaml.safe_load(stream)
+                    except (OSError, yaml.YAMLError) as e:
+                        logger.debug(f"Failed to read YAML file {file_path}: {e}")
```

Comment:
If there is a YAMLERROR, shouldn't we at least warn?  In the current code doesn't this throw cause the application to exit with an error?